src/main/java/org/folio/edge/core/EdgeVerticleHttp.java

```diff
@@ -67,27 +67,27 @@ protected void handleHealthCheck(RoutingContext ctx) {
   }
 
   private void configureSslIfEnabled(HttpServerOptions serverOptions) {
```

Comment:
can we make this utils and giving ablity to dependant modules to use this method to configure ssl
There is two modules can benefit from that by reducing duplicatoin
edge-sip2, edge-connix

Reply:
Introduced SslConfigurationUtils that you can use in modules that do not extend EdgeVerticleHttp